docs/env.md

```diff
@@ -3,28 +3,28 @@
+| REACT_APP_STRIPE_PUBLISHABLE_KEY          | Stripe publishable API key for generating tokens with Stripe API. Use test key (prefix pk*test*) for development.       |
+| REACT_APP_SHARETRIBE_MARKETPLACE_CURRENCY | The currency used in the Marketplace as ISO 4217 currency code. For example: USD, EUR, CAD, AUD, etc.                   |
+| REACT_APP_CANONICAL_ROOT_URL              | Canonical root url of the marketplace. Needed for social media sharing and SEO optimization.                            |
+| NODE_ENV                                  | Node env. Use 'development' for development and 'production' for production.                                            |
+| PORT                                      | Port for server to accept connections.                                                                                  |
+| REACT_APP_ENV                             | A more fine grained env definition than NODE_ENV. Is used for example to differentiate envs in logging.                 |
+| REACT_APP_SHARETRIBE_USING_SSL            | Redirect HTTP to HTTPS?                                                                                                 |
+| SERVER_SHARETRIBE_TRUST_PROXY             | Set when running the app behind a reverse proxy, e.g. in Heroku.                                                        |
+| REACT_APP_PUBLIC_SENTRY_DSN               | See: [Error logging with Sentry](./sentry.md)                                                                           |
+| SERVER_SENTRY_DSN                         | See: [Error logging with Sentry](./sentry.md)                                                                           |
+| REACT_APP_CSP                             | See: [Content Security Policy (CSP)](./content-security-policy.md)                                                      |
+| BASIC_AUTH_USERNAME                       | Set to enable HTTP Basic Auth                                                                                           |
+| BASIC_AUTH_PASSWORD                       | Set to enable HTTP Basic Auth                                                                                           |
+| REACT_APP_GOOGLE_ANALYTICS_ID             | See: [Google Analytics](./analytics.md)                                                                                 |
+| REACT_APP_AVAILABILITY_ENABLED            | Enables availability calendar for listings.                                                                             |
+| REACT_APP_DEFAULT_SEARCHES_ENABLED        | Enables default search suggestions in location autocomplete search input.                                               |
+| REACT_APP_SHARETRIBE_SDK_BASE_URL         | The base url to access the Sharetribe Flex Marketplace API. FTW uses the correct one by default so no need to set this. | 
```

Comment:
Should this line be removed from here altogether? It just confuses FTW users and can be communicated in some other manner to FTW developers.

Reply:
IRL: lets keep for now